doc/manual/src/language/advanced-attributes.md

```diff
@@ -2,256 +2,268 @@
+the file. For example,
+
+```nix
+fetchurl {
+  url = "http://ftp.gnu.org/pub/gnu/hello/hello-2.1.1.tar.gz";
+  sha256 = "1md7jsfd8pa45z73bz1kszpp01yw6x5ljkjk2hx7wl800any6465";
+}
+```
+
+It sometimes happens that the URL of the file changes, e.g., because
+servers are reorganised or no longer available. We then must update
+the call to `fetchurl`, e.g.,
+
+```nix
+fetchurl {
+  url = "ftp://ftp.nluug.nl/pub/gnu/hello/hello-2.1.1.tar.gz";
+  sha256 = "1md7jsfd8pa45z73bz1kszpp01yw6x5ljkjk2hx7wl800any6465";
+}
+```
+
+If a `fetchurl` derivation was treated like a normal derivation, the
+output paths of the derivation and *all derivations depending on it*
+would change. For instance, if we were to change the URL of the
+Glibc source distribution in Nixpkgs (a package on which almost all
+other packages depend) massive rebuilds would be needed. This is
+unfortunate for a change which we know cannot have a real effect as
+it propagates upwards through the dependency graph.
+
+For fixed-output derivations, on the other hand, the name of the
+output path only depends on the `outputHash*` and `name` attributes,
+while all other attributes are ignored for the purpose of computing
+the output path. (The `name` attribute is included because it is
+part of the path.)
+
+As an example, here is the (simplified) Nix expression for
+`fetchurl`:
+
+```nix
+{ stdenv, curl }: # The curl program is used for downloading.
+
+{ url, sha256 }:
+
+stdenv.mkDerivation {
+  name = baseNameOf (toString url);
+  builder = ./builder.sh;
+  buildInputs = [ curl ];
+
+  # This is a fixed-output derivation; the output must be a regular
+  # file with SHA256 hash sha256.
+  outputHashMode = "flat";
+  outputHashAlgo = "sha256";
+  outputHash = sha256;
+
+  inherit url;
+}
+```
+
+The `outputHashAlgo` attribute specifies the hash algorithm used to
+compute the hash. It can currently be `"sha1"`, `"sha256"` or
+`"sha512"`.
+
+The `outputHashMode` attribute determines how the hash is computed.
+It must be one of the following two values:
+
+-   `"flat"`\
+    The output must be a non-executable regular file. If it isn’t,
+    the build fails. The hash is simply computed over the contents
+    of that file (so it’s equal to what Unix commands like
+    `sha256sum` or `sha1sum` produce).
+
+    This is the default.
+
+-   `"recursive"`\
+    The hash is computed over the NAR archive dump of the output
+    (i.e., the result of [`nix-store --dump`](../command-ref/nix-store.md#operation---dump)). In
+    this case, the output can be anything, including a directory
+    tree.
+
+The `outputHash` attribute, finally, must be a string containing
+the hash in either hexadecimal or base-32 notation. (See the
+[`nix-hash` command](../command-ref/nix-hash.md) for information
+about converting to and from base-32 notation.)
+
+## \_\_contentAddressed {#adv-attr-__contentAddressed}
+If this}
+
+&#x20;attribute is set to true, then the derivation
```

Comment:
Looks like the conversion mangled this (should be `If this **experimental** attribute...`).